src/utils/components/withErrorList.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import withErrorList, { getErrorId } from './withErrorList';
+import ErrorList from '../../forms/ErrorList';
+import { mount } from 'enzyme';
+
+const MOCK_FIELD_ID = 'name';
+const MOCK_ERROR_MSG = 'You are not Keith Hernandez';
+
+const TestComponent = (props) => (<h1 {...props}>Hello world</h1>);
+const TestComponentWithErrorList = withErrorList(TestComponent);
+
+describe('error id', () => {
+	it('correctly generates id for `aria-describedby` from field `id` prop', () => {
+		const actual = getErrorId(MOCK_FIELD_ID);
+		const expected = 'name-error';
+		expect(actual).toBe(expected);
+	});
+});
+
+describe('withErrorList wrapped component', () => {
+
+	it('passes `error` to the `ErrorList`', () => {
+		const component = mount(
+			<TestComponentWithErrorList
+				id={MOCK_FIELD_ID}
+				error={MOCK_ERROR_MSG}
+			/>
+		);
+		expect(component.find('li').text()).toBe(MOCK_ERROR_MSG);
+	});
+
+	it('provides TRUE `aria-invalid` prop when wrapped component has `error`', () => {
+		const component = mount(
```

Comment:
I think all of these `mount`s could be `shallow`, and you might be able to consolidate a few tests with a snapshot or two